virt/virtual_machines/vm_networking/virt-attaching-vm-multiple-networks.adoc

```diff
@@ -1,71 +1,40 @@
 :_content-type: ASSEMBLY
-[id="attaching-to-multiple-networks"]
-= Attaching a virtual machine to multiple networks
+[id="virt-attaching-vm-multiple-networks"]
+= Configuring a virtual machine for a Linux bridge network
 include::_attributes/common-attributes.adoc[]
-:context: virt-attaching-multiple-networks
+:context: virt-attaching-vm-multiple-networks
 
 toc::[]
 
-You can attach virtual machines to multiple networks by using Linux bridges.
-You can also import virtual machines with existing workloads that depend on access to multiple interfaces.
+By default, {VirtProductName} is installed with a single, internal pod network.
 
-To attach a virtual machine to an additional network:
+You must create a Linux bridge network attachment definition (NAD) in order to connect to external networks. Otherwise, your network interface cards will be unavailable.
```

Comment:
```suggestion
You must create a Linux bridge network attachment definition (NAD) in order to connect to additional networks. Otherwise, your network interface cards will be unavailable.
```

Reply:
Done.